server/pages/index.pug

```diff
@@ -226,6 +226,56 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
+		script.
+			(function () {
+				// Mini tracks
+				function recordTracksEvent( eventName ) {
+					var statsPixel = 'http://pixel.wp.com/t.gif';
+					var stats = {
+						_pf: navigator.platform,
+						_lg: navigator.language,
+						_en: eventName,
+						_ht: window.screen.height,
+						_wd: window.screen.width,
+						_ut: 'anon',
+						_dl: document.location.origin + '/' + document.location.pathname,
+						_ts: Date.now(),
+						_tz: new Date().getTimezoneOffset() / 60
+					};
+
+					if ( window.currentUser ) {
+						stats[ '_ut' ] = 'wpcom:user_id';
+						stats[ '_ul' ] = currentUser.username;
+						stats[ '_ui' ] = currentUser.ID;
+					}
+
+					var pixel = document.createElement( 'img' );
+					pixel.src = statsPixel + '?' + Object.keys( stats ).map( key => key + '=' + encodeURIComponent( stats[ key ] ) ).join( '& ' );
+
+					var head = document.getElementsByTagName( 'head' )[ 0 ];
```

Comment:
Tracks appends these to the body instead of the head. AFAIK img isn't a valid sub-element of head. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head  

Not sure it has any practical issues, but worth poking at.